### DISCUSSION TITLE
VC firmware update

### DISCUSSION THREAD

**Discussion by DipeshChunilal** [Mr-MIBonk/M.I.B._More-Incredible-Bash#233] [Q&A]
Hi all. Hope all well, thanks for the amazing job helping people around the world.
I recently updated my A4 B9 2017 black edition MMI Plus firmware, M.I.B patches for nav and also POI with speed cameras all thanks to the community here.
Now thinking of updating Virtual Cockpit and before I commit wanted to know if I am picking the right software from mib for my car?

current firmware: MHI2Q_ER_AUG22_P5092 MU1329 

1st picture is my car vc details and second picture from mib solution is the software in question for my car?
![IMG_2807](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/134564640/971abc7b-8f21-4623-ae5a-c7c03d88b853)
![IMG_2814](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/134564640/df6122d6-44f7-4498-a4ad-929d3dedb54f)

**Comment by harman-f** [comment]
Not my core topic

If you do not know about any new great features from that update, better leave it as it is ;)

**Reply by DipeshChunilal** [reply]
I am sorry if I am missing a point. I asked a question here so I could get help or be informed. What features that I do not know about from the update I had? MMI update is separate from vc update

**Reply by harman-f** [reply]
I do not know about this VC update. Hinting, that it most likely does not bting any visible improvements.

My recommendation: leave it as it is ;)

**Reply by DipeshChunilal** [reply]
Thank you much appreciated.

**Comment by neoalius** [comment]
From where can i download this VC firmware ..please

**Reply by Skr1pt** [reply]
https://mibsolution.one/#/1/9/VC-AID%20Updates
